partcad/src/partcad/runtime_python.py

```
@@ -38,12 +57,23 @@ def __exit__(self, *_args):
+        # Setup lock for given venv.
+        # If venv is None then use default (no venv)
+        if venv is None:
+            self.venv = f"{runtime.sandbox}.default"
+            self.venv_lock_name = f".{self.venv}.lock"
+        else:
+            self.venv = venv
+            self.venv_lock_name = f".{runtime.sandbox}.{venv}.lock"
 
     async def __aenter__(self, *_args):
         await sync_threads.run_detached(self.runtime.venv_locks_lock.acquire)
         if not self.venv in self.runtime.venv_locks:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)